### PR TITLE
11.0 purchase order line sequence

### PR DESCRIPTION
**Pull request by CGenie**
This is a copy of `sale-workflow/sale_order_line_sequence` and adjusted for purchases.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@aheficent didn't you have something similar?

File: purchase_order_line_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+{
+    "name": "Purchase Order Line Sequence",
+    "summary": "Propagates PO line sequence to invoices and stock picking.",
+    "version": "11.1.1.0.0",
+    "author": "Eficent, "
+              "Serpent CS, "
+              "Odoo Community Association (OCA)",
+    "category": "Purchase",
+    "website": "https://www.eficent.com/",
```
Comment (nikul-serpentcs) [inline purchase_order_line_sequence/__manifest__.py:14]: @CGenie  URL should be `https://github.com/OCA/purchase-workflow`

File: purchase_order_line_sequence/views/report_purchaseorder.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_purchaseorder_document_sequence" inherit_id="purchase.report_purchaseorder_document">
+        <xpath expr="//table[@class='table table-condensed']/thead/tr/th[1]" position="before">
```
Comment (nikul-serpentcs) [inline purchase_order_line_sequence/views/report_purchaseorder.xml:4]: Use `hasclass` in xpath

File: purchase_order_line_sequence/views/report_purchaseorder.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+            <th><strong>Sequence</strong></th>
+        </xpath>
+        <xpath expr="//table[@class='table table-condensed']/tbody/tr[1]/td[1]" position="before">
```
Comment (nikul-serpentcs) [inline purchase_order_line_sequence/views/report_purchaseorder.xml:7]: same here

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
nitpicking

**Comment by AaronHForgeFlow** [comment]
The module exists in v 10.0: https://github.com/OCA/purchase-workflow/tree/10.0/purchase_order_line_sequence

There is even a PR to v12: https://github.com/OCA/purchase-workflow/pull/613

Can you please migrate from one of those?

**Pull request closed without merging**